cmd/platform/server.go

```diff
@@ -28,9 +28,10 @@ const (
 var MaxNotificationsPerChannelDefault int64 = 1000000
 
 var serverCmd = &cobra.Command{
-	Use:   "server",
-	Short: "Run the Mattermost server",
-	RunE:  runServerCmd,
+	Use:          "server",
+	Short:        "Run the Mattermost server",
+	RunE:         runServerCmd,
+	SilenceUsage: true,
```

Comment:
`SilentUsage: true` simply avoids the command usage to be printed below the error message when a startup error occurs.

It doesn't affect other behaviors: `platform server --help` and `platform server --xxx-invalid-option` still display the command usage properly.